extensions/ql-vscode/src/quick-query.ts

```diff
@@ -70,6 +70,16 @@ function getQuickQueriesDir(ctx: ExtensionContext): string {
   return queriesPath;
 }
 
+
+function updateQuickQueryDir(queriesDir: string, index: number, len: number) {
```

Comment:
Why is this required to be hoisted? I'm not really against it, just curious.

Reply:
There was a linting warning to avoid nested functions. It's not really a big deal and I don't feel strongly about it. It was easier for me to change the code than to change the rule.

Reply:
Actually, now that I look at the rule more closely, it specifically targets named functions that are not in the top-level of another function or global. I think this is generally good since it makes things harder to read if there are lots of nested functions scattered inside of if statements and elsewhere. Made a little fix here.